enhancements/storage/csi-migration.md

```diff
@@ -133,12 +136,36 @@ spec:
 ```
 
 
-Once CSI migration of all storage plugins reaches GA in upstream, the associated feature gates will be enabled by default in OCP. As a result, we will remove the carry-patch we introduced in OCP 4.8. Also, users will not have to use the **TechPreviewNoUpgrade** *FeatureSet* anymore.
-
-### Risks and Mitigations
+#### Risks and Mitigations
 
 A carry-patch means that OCP will be the only Kubernetes distribution exercising this code path, which can lead us to bugs that were not seen anywhere. However, we are confident that the patch is small and self-contained enough to be used.
 
+### GA
+
+Once CSI migration of a given storage plugin reaches GA in upstream, the associated feature gates will be enabled by default in OCP. As a result, we will remove the carry-patch we introduced in OCP 4.8. Also, users will not have to use the **TechPreviewNoUpgrade** *FeatureSet* anymore.
+
+In addition to that, the CSI *StorageClass* (SC) will be set as default on new clusters, but we will not change the default SC on existing clusters. The main reason for that is that users might have set a different quota on the CSI driver SC. An extra benefit of this approach is that we will not override a default SC that was set by the user.
+
+In order to achieve that, these are the main changes introduced in OCP 4.11 with regards to SCs:
+
+* The *defaultstorageclasscontroller* from CSO will be completely removed and we will not create any new in-tree SCs.
+  * SCs that were previously created by this controller will not be owned by any operator and may very well be deleted by the user.
+* A new special-purpose controller will be added to the CSI Driver Operator.
+  * This controller will make sure that a CSI SC always exists.
+  * And it will set that SC as the default one when appropriate.
+  * Also, it will make sure there is only **one** default SC.
+
+In a more detailed way, this is how the new controller will operate and determine whether the CSI SC should be the default one:
+
+* On every sync, this controller fetches all existing SCs.
+* If there are **no** pre-existing **default** SCs in the cluster, then it creates a CSI SC **and** set it as the default.
+  * Otherwise, create the CSI SC but do not make it default.
+* If there are pre-existing SCs in the cluster and more than one is set as default, it will set an appropriate message in the CR asking the user to make amendments.
```

Comment:
cc @RomanBednar, we did not discuss this part. But it would be a trivial addition to the controller that you're going to write.

Reply:
@jsafrane noted, thanks